wolfcrypt/src/integer.c

```diff
@@ -4223,6 +4223,10 @@ static int mp_div_d (mp_int * a, mp_digit b, mp_int * c, mp_digit * d)
       q.used = a->used;
       q.sign = a->sign;
   }
+  else {
+      mp_init(&q); /* initialize to help static analysis */
```

Comment:
Even so, we need to match mp_init() and mp_clear().  Just because XMALLOC no longer allocates memory, what if some version in the future allocates some?